source/documentation/administration_guide/topics/Preparing_GPU_Passthrough.adoc

```diff
@@ -27,7 +27,7 @@ If the host is attached to the {engine-name} already, ensure you place the host
 pci-stub.ids=10de:13ba,10de:0fbc
 ----
 +
-. Blacklist the corresponding drivers on the host. For example, to blacklist nVidia's nouveau driver, next to _pci-stub.ids=xxxx:xxxx_, enter *rdblacklist=nouveau*.
+. Denylist the corresponding drivers on the host. For example, to denylist nVidia's nouveau driver, next to _pci-stub.ids=xxxx:xxxx_, enter *rdblacklist=nouveau*.
```

Comment:
Using "denylist" as a verb looks odd, even though we use 'blacklist". Probably just used to seeing blacklist. I recommend using it as a noun: 

"Add the corresponding driver to the denylist by entering `rdblacklist=<driver>` after the driver ID:"

In the code block example, change nouveau to <driver> or <driver_name>. I don't think the user needs a specific example.